app/src/spIP/java/org/dhis2/usescases/sms/domain/usecase/SendSmsUseCase.kt

```diff
@@ -0,0 +1,79 @@
+package org.dhis2.usescases.sms.domain.usecase
+
+import org.dhis2.usescases.sms.data.model.MessageTemplate
+import org.dhis2.usescases.sms.domain.model.sms.Message
+import org.dhis2.usescases.sms.domain.model.sms.SmsResult
+import org.dhis2.usescases.sms.domain.repository.message.MessageTemplateRepository
+import org.dhis2.usescases.sms.domain.repository.patient.PatientRepository
+import org.dhis2.usescases.sms.domain.repository.preferred.PreferredLanguageRepository
+import org.dhis2.usescases.sms.domain.repository.sms.SmsRepository
+import javax.inject.Inject
+
+const val LANGUAGE_EN = "en"
+
+class SendSmsUseCase @Inject constructor(
```

Comment:
It should not have changes in the use case because the logic is the same, please maintain the file equal to version in 2.9.1. Only change functions to suspend if required